clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1369,6 +1369,64 @@ def paths_to_bids(
     return output_file_treated
 
 
+def _check_existing_images(
```

Comment:
It feels like this function is doing some kind of validation because of the `check` part of the name while it is actually getting files having one of the provided suffixes in their name.
Maybe renaming it to something like `_get_files_with_suffix` or `_filter_files_with_suffix` ? WDYT ?